shaders/MicrobeBackground.gdshader

```diff
@@ -10,6 +10,12 @@ uniform sampler2D layer1 : source_color;
 uniform sampler2D layer2 : source_color;
 uniform sampler2D layer3 : source_color;
 
+// the noise textures, distortion amount, and distortion speed for the distortion shader
+uniform sampler2D noise_tex_1;
+uniform sampler2D noise_tex_2;
```

Comment:
Without also a change to the microbe background godot scene, this PR shouldn't be able to work correctly as these textures are both empty. Also we use camelCase naming convention in Thrive so these new variables you've added should be named `noiseTex1`, `distortionFactor` etc. rather than be named with underscores.

Reply:
Confirmed that it doesn't work, and will change the naming convention. 

About the microbe background scene - is there a specific .tscn for the microbe background? I'm trying to find it on godot, but can only find microbestage.tscn and this background shader file.

Reply:
The background is in the microbe camera scene as the background is attached to the camera (as that's how it stays always visible):

![kuva](https://github.com/user-attachments/assets/66167a12-a905-4333-8806-cad0e3d6260b)